magento/magento2
# Removed extra qty field from bundle configuration

### Description (*)

This PR will remove the duplicate qty field from the bundle configure modal whilst creating order from the backend
### Related Pull Requests
https://github.com/magento/inventory/pull/3254
https://github.com/magento/partners-magento2b2b/pull/513


### Fixed Issues (if relevant)

1. Fixes magento/magento2#30306

### Manual testing scenarios (*)

1. Create a bundle product
2. Create a new order from admin
3. Add the bundle product to the order
4. Click "Configure" link of the bundle product in the "Please select products" grid
5. Configure Product modal opens
6. Check the Quantity field

### Expected Result
![image](https://user-images.githubusercontent.com/39480008/96995996-bb5c0580-154c-11eb-8456-ace18fd55a14.png)


### Questions or comments


### Contribution checklist (*)
